### PR TITLE
Backport #1120 - Ceph health warn trigger modification

### PR DESCRIPTION
**Pull request by kelvk**
Backport of #1120 - Classify alerts into blocked op and non-blocked op, as well as providing an informational flag for identifying VIP holder. This is only a client side change and cluster must be upgraded to 6.0.0 or master for #1120 to take effect.

### CODE REVIEW

**Comment by erhudy** [comment]
Can you bump the version to 5.13.7?

**Comment by kelvk** [comment]
Bumped

File: cookbooks/bcpc/recipes/zabbix-agent.rb [hunk 1]
```diff
@@ -48,7 +48,6 @@
     %w{zabbix-agent zabbix-get zabbix-sender}.each do |zabbix_package|
       package zabbix_package do
         action :upgrade
-        options '-o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold"'
```
Comment (erhudy) [inline cookbooks/bcpc/recipes/zabbix-agent.rb]: Was this line intentionally removed?

Comment (kelvk) [inline cookbooks/bcpc/recipes/zabbix-agent.rb]: No. Restored.

**Pull request merged**